internal/resource-request-operator/resourceRequest_operator_test.go

```diff
@@ -362,12 +386,55 @@ var _ = Describe("ResourceRequest Operator", func() {
 				}
 				return true
 			}, timeout, interval).Should(BeTrue())
+			By("Checking ResourceOffer update")
```

Comment:
The By clauses in the test should be more explicit about the expected outcome of the test.